cms/envs/aws_migrate.py

```diff
@@ -1,30 +0,0 @@
-"""
```

Comment:
@omarkhan Please look at the current state of aws_migrate.py and update this change so that we target the correct databases (and not just default).